python/pfs/utils/constants.py

```diff
@@ -0,0 +1,18 @@
+"""General PFS engineering constants
+"""
+N_SPECTROGRAPHS = 4  # Number of spectrographs
```

Comment:
In theory this is a good idea, but in practice `4` is hardwired in many places (e.g. ingest.py, in `PfsParseTask`) and N_SPECTROGRAPHS isn't used anywhere.  Are you planning to replace all other occurrences of the number of spectrographs?   If not, this seems like false security to me.

Reply:
Yes, my general intention is to introduce a centralized set of constants across the PFS.

Reply:
Yes, my intention is to create a centralized list of constants used by PFS code to avoid potential consistences across the software, so other places where it's hardwired will be updated accordingly (in separate tickets).